src/main/resources/templates/home.html

```diff
@@ -11,78 +11,40 @@
       </th:block>
 
       <div class="grid-row">
-
-        <main class="column-two-thirds" id="main" role="main">
-          <h1 class="heading-large" th:utext="${friendlyRegisterName}"></h1>
-          <th:block th:utext="${registerText}"></th:block>
-          <p data-click-events="" data-click-category="Content" data-click-action="Button Clicked">
+        <div class="column-two-thirds">
+          <h1 class="heading-large">
+            <th:block th:utext="${friendlyRegisterName}"></th:block>
+            <span class="heading-secondary" th:utext="${registerText}"></span>
+          </h1>
+          <span data-click-events="" data-click-category="Content" data-click-action="Button Clicked">
             <a href="/records" class="button" role="button">View Register</a>
-          </p>
-          <p>The <span th:utext="${friendlyRegisterName}"></span> contains <span th:text="${totalRecords}"></span>
-            records and includes the following fields:</p>
-          <div th:replace="fragments/field-list.html::field-list(${fields})"></div>
+          </span>
+
+          <h2 class="heading-medium">Using this register</h2>
 
-          <h2>Using this register</h2>
+          <h3 class="heading-small">Base URL</h3>
+          <pre class="preview-box"><code class="language-markup" th:utext="${registerResolver.baseUriFor(registerName)}"></code></pre>
+
+          <h3 class="heading-small">Endpoints</h3>
           <p th:replace="fragments/download-and-preview-home.html::download-and-preview-home"></p>
-          <p>Each register has an open API you can use to access the data without any authentication. There's more
-            information about using the register APIs in the <a th:href="${homepageContent.techDocsUrl}"
-                                                                target="_blank">technical documentation</a>.</p>
-          <p>If you're building a service, you can use this data to create elements for forms.</p>
-          <p>You might need to combine data from this register with data from another source to do this.</p>
-        </main>
+
+          <p><code class="code">/records</code> returns the most recent item for every key in a register.</p>
+
+          <p><code class="code">/entries</code> returns the log of all entries appended to a register and the hash of the item attached to each entry, ordered by entry number.</p>
+
+          <p><code class="code">/item{hash}</code> returns an item is a blob of data. Each item in a register is linked to an entry by a unique hash. The entry describes when the item was added to the register.</p>
```

Comment:
Should be _returns an item **as** a blob of data_.

Also, the endpoint should be `/item/{hash}`, and an example might be useful to specify that the prefix will be `sha-256:`

Reply:
This text is being reviewed by @michaelabenyohai